### PR TITLE
Meetup notes 05-10-2018

### PR DESCRIPTION
**Pull request by rafael**
### Description

Meetup notes from 05-10-2018.

### CODE REVIEW

File: doc/meetups_notes/05-10-2018.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### Vitess 3.0 Plan
+
+* Finish full subquery support.
+* Multi shard DML’s support of PR’s that are already opened.
+* Getting all the docs done is  too big of a project, for the next release we will go through all the pull requests that have been merged and create a release note based on that. 
+* Once Planet Scale has a team in place, cadence for releases should be every couple months. 
+
+### Planet scale update
+
+* Most of the requests that planet scale it’s getting are around helping people deploy vitess in their clusters. In the short in term this what PlanetScale will be focused on.
```
Comment (sougou) [inline doc/meetups_notes/05-10-2018.md]: `that planet scale is getting`

File: doc/meetups_notes/05-10-2018.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+* Finish full subquery support.
+* Multi shard DML’s support of PR’s that are already opened.
+* Getting all the docs done is  too big of a project, for the next release we will go through all the pull requests that have been merged and create a release note based on that. 
+* Once Planet Scale has a team in place, cadence for releases should be every couple months. 
+
+### Planet scale update
+
+* Most of the requests that planet scale it’s getting are around helping people deploy vitess in their clusters. In the short in term this what PlanetScale will be focused on. 
+* In the open source realm, PlanetScale will building and hosting binaries for vitess.
```
Comment (sougou) [inline doc/meetups_notes/05-10-2018.md]: `will build and host`

File: doc/meetups_notes/05-10-2018.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* Multi shard DML’s support of PR’s that are already opened.
+* Getting all the docs done is  too big of a project, for the next release we will go through all the pull requests that have been merged and create a release note based on that. 
+* Once Planet Scale has a team in place, cadence for releases should be every couple months. 
+
+### Planet scale update
+
+* Most of the requests that planet scale it’s getting are around helping people deploy vitess in their clusters. In the short in term this what PlanetScale will be focused on. 
+* In the open source realm, PlanetScale will building and hosting binaries for vitess. 
+* Hiring process is ongoing, offers out and interviews are happening.
+*First engineer starting on June first.
```
Comment (sougou) [inline doc/meetups_notes/05-10-2018.md]: Space between `*` and `F`

**Comment by acharis** [comment]
thanks!

**Pull request merged**